target/linux/mediatek/filogic/config-6.6

```diff
@@ -225,6 +225,8 @@ CONFIG_IRQ_TIME_ACCOUNTING=y
 CONFIG_IRQ_WORK=y
 CONFIG_JBD2=y
 CONFIG_JUMP_LABEL=y
+CONFIG_LEDS_LP55XX_COMMON=y
+CONFIG_LEDS_LP5523=y
```

Comment:
Please move this into a kernel package, see b33fa6ac9065028046016ea631f6be0926d7abd0.
This allows you to add it only to one board and not for all.